journals/apps/journals/models.py

```diff
@@ -176,20 +176,37 @@ def __str__(self):
         return str(self.uuid)
 
     @classmethod
-    def user_has_access(cls, user, journal):
-        """ Checks if the user has access to this journal """
+    def get_user_accessible_journals(cls, user):
+        """ Finds all journals that user has access to """
         if user.can_access_admin:
-            return True
+            return list(Journal.objects.all())
```

Comment:
per above, just return `Journal.objects.all()`, no need to make it a list

Reply:
Fixed